apps-rendering/src/components/HeadlineTag/index.tsx

```diff
@@ -24,7 +29,9 @@ type Props = {
 };
 
 const HeadlineTag: FC<Props> = ({ tagText, format }) => (
-	<span css={headlineTagStyles(format)}>{tagText}</span>
+	<div css={headlineTagWrapper}>
```

Comment:
I'm just wondering, what happens if this component is used in other places? Do we need `articleWidthStyles` for all diferent use cases?

Reply:
I can't imagine when we wouldn't - the only other place HeadlineTag is used atm is Interviews and InterviewHeadline sets the width explicitly (perhaps this should also use articleWidthStyles instead - but didn't want to change too much)

Reply:
I think we always want the HeadlineTag to align with the rest of the article and I think that's what using articleWidthStyles here achieves? But I could be wrong - would any issues be highlighted in chromatic tests?

Reply:
Yea you're right. It may be better if the interview also uses the articleWidthStyles. But I think right now there are 2 layers of padding applied. One in `InterviewHeadline` and one in `HeadlineTag`

Reply:
Maybe in the `HeadlineTag` we cvan just apply this spacing for when it's `Analysis`?